api/inc/api.h

```diff
@@ -70,6 +70,8 @@ typedef struct {
     uvisor_pool_slot_t (*pool_queue_dequeue_first)(uvisor_pool_queue_t *);
     uvisor_pool_slot_t (*pool_queue_find_first)(uvisor_pool_queue_t *, TQueryFN_Ptr, void *);
 
+    void (*vmpu_mem_invalidate)(void);
```

Comment:
Isn't the offset of the function here different from the one in https://github.com/ARMmbed/uvisor/pull/376/commits/ecb88f98dc439ab10d070a2203b5feae189ec263#diff-a763365abbc3765a4810f65eec790b53R48 ?